### PR TITLE
feat(integrations): utility to load keras models from a W&B artifact

### PR DESCRIPTION
**Pull request by soumik12345**
Fixes [GROWTH2-292](https://wandb.atlassian.net/browse/GROWTH2-292)

Description
-----------
This PR adds a utility function `load_model_from_artifact` that allows us to load a Keras model(s) directly from a W&B artifact.

Collaboration with @ayulockin 

Testing
-------
How was this PR tested?

Checklist
-------
- [ ] Include reference to internal ticket "Fixes WB-NNNN" and/or GitHub issue "Fixes #NNNN" (if applicable)
- [ ] Ensure PR title compliance with the [conventional commits standards](https://github.com/wandb/wandb/blob/main/CONTRIBUTING.md#conventional-commits)

[GROWTH2-292]: https://wandb.atlassian.net/browse/GROWTH2-292?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: wandb/integration/keras/utils.py [hunk 1]
```diff
@@ -0,0 +1,75 @@
+import os
```
Comment (parambharat) [inline wandb/integration/keras/utils.py:1]: Add module level docstring

File: wandb/integration/keras/utils.py
Comment (parambharat) [inline wandb/integration/keras/utils.py:1]: - Run isort to sort imports

**Review by parambharat — COMMENTED** [review]
- Add functional tests to test the model loading with different argument settings. 
- Maybe consider add the ability to load model weights from artifacts where `save_weights_only` is true.

File: wandb/integration/keras/utils.py (hunk 1 above)
Comment (soumik12345) [reply wandb/integration/keras/utils.py]: @parambharat We don't add module-level docstring for the Keras integration. Is this something specific to this function, or the whole Keras integration module itself?

**Comment by soumik12345** [comment]
> * Maybe consider add the ability to load model weights from artifacts where `save_weights_only` is true.

@parambharat  This would probably require another function named `load_weights_from_artifact`.

File: wandb/integration/keras/utils.py (hunk 1 above)
Comment (parambharat) [reply wandb/integration/keras/utils.py]: Well, It's generally a good practice to have module level docstrings for all python modules. I'll defer to your call on whether you want to have it for just `utils.py` or for the whole keras integration module.

File: wandb/integration/keras/utils.py [hunk 2]
```diff
@@ -37,6 +62,8 @@ def load_model_from_artifact(
             )
 
     if model_filepaths is None:
+        wandb.termlog("loading model")
```
Comment (parambharat) [inline wandb/integration/keras/utils.py:65]: Perhaps, check for verbosity here also before logging.

**Pull request closed without merging**